src/components/Nav.tsx

```diff
@@ -62,6 +62,11 @@ const Nav = ({ network }) => {
                     <A href="/history" onClick={() => setHamburger(false)}>
                         {t("history")}
                     </A>
+                    <Show when={statusUrl}>
+                        <a class="external" target="_blank" href={statusUrl}>
+                            {t("status")}
+                        </a>
+                    </Show>
```

Comment:
Nack
How often do you click a status link? Almost never. Makes no sense to show it to users all the time

Reply:
agree

Reply:
plus the web app will scream in your face if sth is not working anyways.